controller/posts-controller.js

```diff
@@ -0,0 +1,72 @@
+const Comment = require("../models/comment");
+const Posts = require("../models/posts");
+const User = require("../models/user");
+
+const validation = require("./validation");
+
+exports.addPost = async (req, res) => {
+  const { title, description } = req.body;
+  const userId = req.user._id;
+  let newPost = new Posts({
```

Comment:
Instead of  `newPost` you can just use `post` here.

Reply:
updated